javascripts/search.js

```diff
@@ -201,12 +201,12 @@ search.filter('description', function($sce) {
   }
 });
 
-search.filter('question', function($sce) {
+search.filter('question', function() {
   return function(result) {
     if (result.highlight && result.highlight._source.sys_content_plaintext) {
-      return $sce.trustAsHtml(result.highlight._source.sys_content_plaintext[0]);
+      return String(result.highlight._source.sys_content_plaintext[0]).replace(/&lt;/g,"<").replace(/&gt;/g,">");
     }
-    return $sce.trustAsHtml(result._source.sys_content_plaintext);
+    return String(result._source.sys_content_plaintext).replace(/&lt;/g,"<").replace(/&gt;/g,">");
```

Comment:
OH! Scratch what I said. I was thinking we were trying to remove the tags. This looks like a sticky situation. It almost looks like if we stripped the tags using the code I just posted on the `result._source.sys_content` then it would be valid to use the trustAsHtml function as is. So it would look like 
`return $sce.trustAsHtml(result._source.sys_content.replace(/<[^>]+>/gm, ''));`